generators/app/templates/infra/gcp/Makefile

```diff
@@ -0,0 +1,25 @@
+# Updates the documentation of the README with the resources of the terraform files
```

Comment:
This file is the same for aws and gcp, so I think we should put it in a folder `infra/common` (as well as .gitignore, etc.)